### PR TITLE
Update docs with new fields for web messenger settings

### PR DESCRIPTION
**Pull request by chloepouprom**
Widget badge + more color customization

@lemieux @jpjoyal @alavers @mspensieri @liyefei737

### CODE REVIEW

**Comment by mspensieri** [comment]
👍

**Comment by Mario54** [comment]
👍

**Comment by lemieux** [comment]
`source/images/web_messenger_settings.png` will need to be redone, the remove image link shouldn't be there if there's no image.

otherwise :+1:

**Comment by liyefei737** [comment]
👍

File: source/includes/customization/_styling_conversations_js.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+
+### Basic Color Customization
+
+For basic customization of colors, you can set the Brand Color, Conversation Color and Link Color in the web settings of the Smooch dashboard.
+
+* The **Brand Color** customizes the color of the messenger header. It is also used for the color of the badge or tab in idle state, as well as the color of the default app icon. If no color is specified, the brand color will default to <span style='color: #65758e; font-weight:bold;'>#65758e</span>.
+* The **Conversation Color** customizes the color of customer messages and actions in the footer. If no color is specified, the conversation color will default to <span style='color: #0099ff; font-weight: bold;'>#0099ff</span>.
+* The **Link Color** changes the appearance of links and buttons in your messages. It is also used for the 'Send' button when it is in active state. If no color is specified, the link color will default to <span style='color: #0099ff; font-weight: bold;'>#0099ff<span>.
 
 <img alt="Color Customization" src="../images/color_customization_web.png" />
```
Comment (jpjoyal) [inline source/includes/customization/_styling_conversations_js.md]: Is it link color or action color in the end ?

File: source/includes/customization/_styling_conversations_js.md [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
```
Comment (jpjoyal) [inline source/includes/customization/_styling_conversations_js.md]: Can you also change the `Advanced Customization` section to mention something along the like "If you go down this path, we advice in using NPM or bower installation. As we push updated on the CDN your CSS customization could break. "

File: source/includes/customization/_styling_conversations_js.md (hunk 1 above)
Comment (chloepouprom) [inline source/includes/customization/_styling_conversations_js.md]: Oops, should be action color

File: source/includes/customization/_styling_conversations_js.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ The web messenger settings page allows for customization of various fields.
 
 ### Display Style
 
-The web messenger can be displayed as a badge or as a tab. You can select the style in the web settings of the Smooch dashboard. The default style is the badge mode.
+The web messenger can be displayed as a button or as a tab. You can select the style in the web settings of the Smooch dashboard. The default style is the button mode.
 
-With the badge style web messenger, you have the option of selecting your own badge icon. The image must be at least 120 x 120 pixels and must be in either JPG, PNG, or GIF format. 
+With the button style web messenger, you have the option of selecting your own button icon. The image must be at least 120 x 120 pixels and must be in either JPG, PNG, or GIF format.
```
Comment (lemieux) [inline source/includes/customization/_styling_conversations_js.md]: I think we ask for 200x200 in the webapp.

**Comment by dannytranlx** [comment]
👍

**Comment by alavers** [comment]
:+!

**Pull request merged**